### PR TITLE
Added new 'lsm' section for selecting the desired Linux Security Module during installation

### PR DESCRIPTION
**Pull request by teclator**
For more info, see yast/yast-security#115 and yast/yast-installation#1006

### CODE REVIEW

**Review by jreidinger — APPROVED** [review]

File: control/control.leanos.xml [hunk 1]
```diff
@@ -86,12 +86,22 @@ textdomain="control"
...
+          <!-- Set SELinux disabled mode by default -->
+          <apparmor>
+            <!-- Not configurable because of https://bugzilla.suse.com/show_bug.cgi?id=1184215 -->
+            <patterns>apparmor</patterns>
+          </apparmor>
+          <selinux>
+            <mode>disabled</mode>
+            <!-- Not configurable because of https://bugzilla.suse.com/show_bug.cgi?id=1184215 -->
+            <configurable config:type="boolean">false</configurable>
+            <selectable config:type="boolean">false</selectable>
```
Comment (dgdavid) [inline control/control.leanos.xml:101]: If it is not selectable… I guess configurable and mode makes no sense. What do you think?

Comment (teclator) [reply control/control.leanos.xml:101]: It exposes all the options, we could probably omit it but then why not omit the section completely...

Comment (dgdavid) [reply control/control.leanos.xml:101]: Ok, now makes sense to me. Continue exposing all the options and go ahead!

**Review by dgdavid — APPROVED** [review]
LGTM

**Comment by teclator** [comment]
@dgdavid tested it and I think we should modify the options to also allow to preselect **none** as the selected **LSM** or not select nothing at all.

**Comment by dgdavid** [comment]
> @dgdavid tested it and I think we should modify the options to also allow to preselect **none** as the selected **LSM** or not select nothing at all.

Make sense.

**Pull request merged**